docs/features/kubernetes/configuration.md

```diff
@@ -341,6 +341,7 @@ For example:
 - type: 'gke'
   projectId: 'gke-clusters'
   region: 'europe-west1' # optional
+  authProvider: `google` # optional
```

Comment:
```suggestion
  authProvider: 'google' # optional
```

Reply:
@viglesiasce Can you help me with this change? Please